libllvm-pdb-wrapper/wrapper.cpp

```diff
@@ -101,6 +101,7 @@ bool pdb_file::initialize(bool is_64bit ,uint32_t Age,uint32_t Signature,llvm::c
     InfoBuilder.setHashPDBContentsToGUID(false);
     InfoBuilder.setSignature(Signature);
     InfoBuilder.setGuid(OurGuid);
+    InfoBuilder.addFeature(llvm::pdb::PdbRaw_FeatureSig::VC140);
```

Comment:
copied from here https://github.com/llvm/llvm-project/blob/8ba9ee46e465a56a54f8361703d3af7f4bc98d63/llvm/tools/llvm-pdbutil/llvm-pdbutil.cpp#L1335

Reply:
Do you know what this does?

Reply:
well no but it seems to be important when merging pdbs and comes from this enum
```
enum class PdbRaw_FeatureSig : uint32_t {
  VC110 = PdbImplVC110,
  VC140 = PdbImplVC140,
  NoTypeMerge = 0x4D544F4E,
  MinimalDebugInfo = 0x494E494D,
};
```
so my best guess it tells the pdb how to expect the data.

Reply:
its also seems connected with this data
```
enum PdbRaw_Features : uint32_t {
  PdbFeatureNone = 0x0,
  PdbFeatureContainsIdStream = 0x1,
  PdbFeatureMinimalDebugInfo = 0x2,
  PdbFeatureNoTypeMerging = 0x4,
  LLVM_MARK_AS_BITMASK_ENUM(/* LargestValue = */ PdbFeatureNoTypeMerging)
};
```